.github/workflows/main.yml

```diff
@@ -68,6 +68,8 @@ jobs:
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: image=moby/buildkit:v0.10.0
```

Comment:
@kirill-sizov , please add a comment which explain the change. It will be great to have a link on the issue as well.